joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/OperatorTests.scala

```diff
@@ -733,4 +710,20 @@ class OperatorTests extends PhpCode2CpgFixture {
       absCall.signature shouldBe s"${Defines.UnresolvedSignature}(1)"
     }
   }
+
+  "array/list unpacking should be lowered to several assignments" in {
+    val cpg = code("""<?php
+        |list(list($a, $b), $c) = $arr;
+        |""".stripMargin)
+    // finds the block containing the assignments
+    val block     = cpg.all.collect { case block: Block if block.lineNumber.contains(2) => block }.head
+    val blockCode = block.astChildren.sortBy(_.order).map(_.code).mkString("\n")
```

Comment:
Ideally, the test would look something like

```scala
inside(block.astChildren.assignment.l) {
 case tmp0 :: tmp1 :: tmp2 :: a :: b :: c =>
    tmp0.code shouldBe "$tmp0 = $arr"
    tmp0.source.label shouldBe NodeTypes.Identifier
    tmp0.source.code shouldBe "$arr"
    tmp0.target.label shouldBe NodeTypes.Identifier
    tmp0.target.code shouldBe "$tmp0"
    // etc.
}
```

Reply:
Thanks. I have changed the test in this way